config/initializers/docusign_rest.rb

```diff
@@ -0,0 +1,17 @@
+require 'docusign_rest'
+
+DocusignRest.configure do |config|
+  if Rails.env.test?
```

Comment:
Actually I had tried this, but it wasn't working. You pointing it out made me look a bit harder, and found out the problem was with the version of the dotenv gem we are using, where it would load .env.local with higher priority that .env.test.

Anyways, thanks for pointing it out and pushing me to figure out what was going on 👍